lib/datadog/appsec/contrib/rack/request_body_middleware.rb

```diff
@@ -24,15 +24,15 @@ def call(env)
             # TODO: handle exceptions, except for @app.call
 
             http_response = nil
-            block_actions = catch(::Datadog::AppSec::Ext::INTERRUPT) do
+            block_action_params = catch(::Datadog::AppSec::Ext::INTERRUPT) do
```

Comment:
As a suggestion, we catching interrupt here, do we want to call it `interrupt_params`? Because we might not block, but redirect.

Reply:
fixed [558f96f](https://github.com/DataDog/dd-trace-rb/pull/4300/commits/558f96fff49d40d5e7064f33900cc54b1174704a)